src/contexts/W3iContext/index.tsx

```diff
@@ -169,6 +148,22 @@ const W3iContextProvider: React.FC<W3iContextProviderProps> = ({ children }) =>
     }
   }, [chatClient, refreshChatState])
 
+  useEffect(() => {
+    const handleRegistration = async () => {
+      if (chatClient && userPubkey) {
+        try {
+          const registeredKeyRes = await chatClient.register({ account: `eip155:1:${userPubkey}` })
```

Comment:
Can we create a follow up issue to replace the hardcoded chainId with the current chainId ?

Reply:
We definitely need to, will do